EventFilter/L1TRawToDigi/src/OmtfMuonUnpacker.cc

```diff
@@ -27,8 +27,20 @@ void MuonUnpacker::unpack(unsigned int fed, unsigned int amc, const MuonDataWord
   digi.setTrackAddress(trackAddr);
   digi.setTFIdentifiers(iProcessor, overlap);
   int bx = data.bxNum()-3;
-  LogTrace("")  <<"OMTF Muon, BX="<<bx<<", hwPt="<<digi.hwPt()<< std::endl;
-  if(std::abs(bx) <= 3) producedMuonDigis->push_back(bx,digi);
+  LogTrace("")  <<"OMTF Muon, BX="<<bx<<", hwPt="<<digi.hwPt()<<", link: "<<digi.link() << std::endl;
+
+  // add digi to collection, keep fixed ascending link orderi (insert in proper place)
+  if(std::abs(bx) <= 3) {
```

Comment:
Why only for BX range +/-2? Should it not work the same for all BX?

Reply:
Hi, I have not changed BX range. It is/was [-3,+3].  OMTF in maximum (it depends on settings) stores 8BXes with "3rd" as the trigger one.

Reply:
There are 3 bits for BX in data.